examples/testhandler/tests/test_handler.py

```diff
@@ -137,3 +140,10 @@ def test_close_cache(project):
     versions = handler.cache.counterforVersion.keys()
     assert len(versions) == 0
     assert len(handler.cache.cache) == 0
+
+
+def test_sys_executable(project):
+    """
+    Make sure the current python interpreter is the same as the one used by the compiler
+    """
+    assert Path(project._env_path) == Path(sys.executable).parent.parent
```

Comment:
Looks good to me. You've verified that this test failed without your changes?

Reply:
Yes I did :+1: